subprojects/packagefiles/openssl/meson.build

```diff
@@ -0,0 +1,406 @@
+project(
+  'openssl',
+  'c',
+  version: '1.1.1k',
+  license : 'Apache-2.0',
+  meson_version: '>= 0.55',
+  default_options: [
+    'warning_level=1',
```

Comment:
OpenSSL uses `-DNDEBUG` when building with optimizations, maybe we should use the `b_ndebug=if-release` option?

Reply:
We have already discussed this before: https://github.com/mesonbuild/wrapdb/pull/113#pullrequestreview-728044663

Turns out `b_ndebug` is not supported on subproject level, so we rely on global setup and user awareness here.